File: osp-be/src/main/java/it/fincons/osp/dto/TipoAllegatoGruppoStatoProcessoDTO.java
package it.fincons.osp.dto;

import lombok.Data;

@Data
public class TipoAllegatoGruppoStatoProcessoDTO {
	
	private TipoAllegatoDTO tipoAllegato;

	private boolean flagObbligatorio;

	private boolean flagTestoLibero;

	private Integer idStatoPratica;

	private Integer idTipoProcesso;

	private Integer idGruppo;

}

File: osp-fe/src/app/shared/models/dto/tipo-allegato-gruppo-stato-processo-dto.ts
import { TipoAllegatoDTO } from "./tipo-allegato-dto";

export class TipoAllegatoGruppoStatoProcessoDTO {
	flagObbligatorio: boolean;
	flagTestoLibero: boolean;
	idGruppo: number;
	idStatoPratica: number;
	idTipoProcesso: number;
	tipoAllegato: TipoAllegatoDTO;
}
